__tests__/port.test.js

```diff
@@ -0,0 +1,37 @@
+const Port = require("../src/port");
+
+describe("port", () => {
+  describe("port is instantiated", () => {
+    let port;
+    beforeEach(() => {
+      port = new Port("Dover");
+    });
```

Comment:
Should have an empty line before and after your `beforeEach` to improve readability